public/friendlycode/js/fc/ui/text.js

```diff
@@ -12,23 +12,6 @@ define(["jquery", "fc/prefs", "analytics"], function($, Preferences, analytics)
       return item.length ? item : null;
     }
 
-    Preferences.on("change:textSize", function() {
-      var size = $("li[data-default-size]", menu).attr("data-size");
-      var prefSize = Preferences.get("textSize");
-      if (prefSize && typeof(prefSize) == "string" && menuItem(prefSize))
-        size = prefSize;
-
-      $(codeMirror.getWrapperElement()).attr("data-size", size);
-      codeMirror.refresh();
-
-      // Reparse as well, in case there were any errors.
-      codeMirror.reparse();
-
-      // Mark text size in drop-down.
-      menuItems.removeClass("selected");

```

Comment:
This part is making sure that the menu items reflect which size was selected. Compare your patch with the way it looks/works on [thimble proper](thimble.webmaker.org) to see if we need this put back in.